static/js/event_page/types.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { DisasterType } from "../disaster_dashboard/constants";
+import { NamedTypedPlace } from "../shared/types";
+
+/**
+ * Types specific to Event Page
+ */
+
+export interface EventInfo {
```

Comment:
in general, please add documentation for shared interfaces. 

can we add this type as it's needed?

Reply:
I meant it as a placeholder, removed the interface. Will add types as necessary in future PRs.